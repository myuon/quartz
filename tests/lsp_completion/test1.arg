--line 10 --column 7 --dot