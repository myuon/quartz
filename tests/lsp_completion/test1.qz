struct T {
    a: i32,
    b: i32,
    c: i32,
    d: string,
}

fun main() {
    let t = T { a: 1, b: 2, c: 3, d: "hello" };
    t.
}
