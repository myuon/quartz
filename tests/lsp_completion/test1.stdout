{"items":[{"kind":"field","detail":"i32","label":"c"},{"kind":"field","detail":"string","label":"d"},{"kind":"field","detail":"i32","label":"a"},{"kind":"field","detail":"i32","label":"b"}]}
